CriticalMass/UIFont+DynamicFont.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIFont+Scalable.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 30.05.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import UIKit
+
+extension UIFont {
+    /// Creates a System Font that is reacting to Font size changes from the OS.
+    ///
+    /// - Parameters:
+    ///   - fontSize: Initial font size.
+    ///   - weight: UIFont.Weight of the font.
+    /// - Returns: UIFont of Font SystemFont that is reacting to Dynamic Type changes.
+    static func scalableSystemFont(fontSize: CGFloat, weight: UIFont.Weight) -> UIFont {
```

Comment:
It looks like we have 2 font styles that don't exist in the system font styles:
UIFont.scalableSystemFont(fontSize: 15, weight: .bold)
UIFont.scalableSystemFont(fontSize: 13, weight: .medium)

Maybe we can expose them in a central place and make scalableSystemFont private